addons/mrp/models/stock_warehouse.py

```diff
@@ -88,6 +89,7 @@ def _get_routes_values(self):
                 }
             }
         })
+        routes.update(self._get_receive_routes_values('buy_to_resupply'))
```

Comment:
not sure it's buy_to_resupply in mrp :D

Reply:
Whoops, copy and paste gone terribly wrong. It's just a formality anyways, mrp always triggers the update on its own :)